static/index.html

```diff
@@ -1,90 +1,99 @@
 <!DOCTYPE html>
```

Comment:
We appear to have different HTML formatters :-D

Mine wants to change all the whitespace in this file back.

Reply:
Played around with formatter settings. It looks like I have Prettier as the formatter, but your changes match with VSCode's default formatter.